it/src/test/java/thredds/client/catalog/TestResolve.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 1998-2009 University Corporation for Atmospheric Research/Unidata
+ * Copyright 1998-2015 University Corporation for Atmospheric Research/Unidata
```

Comment:
2017?

Reply:
and remove rest of copyright block?

Reply:
Yeah, I've made no effort whatsoever to update copyright blocks. The only reason this changed is because of a copy/paste, I assume. I think those blocks should be handled by automation, not us.

Reply:
Ideally automation, but that's not in place yet. No need to update now - we'll address that later.